src/Datadog.Trace.ClrProfiler.Managed/Integrations/MongoDbIntegration.cs

```diff
@@ -440,19 +440,9 @@ private static Scope CreateScope(object wireProtocol, object connection)
                         }
                     }
 
-                    // get the "query" element from the command BsonDocument, if it exists
-                    if (command.TryCallMethod("Contains", "query", out bool found) && found)
-                    {
-                        if (command.TryCallMethod("GetElement", "query", out object queryElement) && queryElement != null)
-                        {
-                            if (queryElement.TryGetPropertyValue("Value", out object queryValue) && queryValue != null)
-                            {
-                                query = queryValue.ToString();
-                            }
-                        }
-                    }
+                    query = command.ToString();
 
-                    resourceName = $"{operationName ?? "operation"} {databaseName ?? "database"} {query ?? "query"}";
+                    resourceName = $"{operationName ?? "operation"} {databaseName ?? "database"} {query}";
```

Comment:
What does `query` look like? I'm wondering what the resource name is going to look like after concatenating these. Does it already include the operation or database name?

Reply:
It looks like we don't use `collectionName` anywhere so we can skip getting that value.

Reply:
Examples of resource name: 
```
find test-db { "find" : "employees", "filter" : { "a" : 1 } }
getLastError admin { "getLastError" : 1 }
isMaster admin { "isMaster" : 1, "client" : { "driver" : { "name" : "mongo-csharp-driver", "version" : "2.7.2.0" }, "os" : { "type" : "Windows", "name" : "Microsoft Windows 10.0.19041", "architecture" : "x86_64", "version" : "10.0.19041" }, "platform" : ".NET 5.0.1-servicing.20575.16" } }
```

The operation name is redundant with the query, I can remove it if you prefer.

Reply:
> It looks like we don't use `collectionName` anywhere so we can skip getting that value above.

collectionName is assigned to a tag further down:

```csharp
tags.Collection = collectionName;
```

Reply:
CTRL+F failed me 🤦🏽